caffe2/CMakeLists.txt

```diff
@@ -115,7 +115,10 @@ add_library(caffe2 ${Caffe2_CPU_SRCS})
 target_link_libraries(caffe2 PUBLIC caffe2_protos ${Caffe2_PUBLIC_DEPENDENCY_LIBS})
 target_link_libraries(caffe2 PRIVATE ${Caffe2_DEPENDENCY_LIBS})
 target_link_libraries(caffe2 PRIVATE ${Caffe2_DEPENDENCY_WHOLE_LINK_LIBS})
-target_include_directories(caffe2 INTERFACE $<INSTALL_INTERFACE:include>)
+target_include_directories(caffe2
```

Comment:
This looks strange, what are you trying to do?

Reply:
it is a way to get files inside `caffe2/core/nomnigraph/` to have `caffe2/core/nomnigraph/include` in their header namespace

Reply:
I've simplified the build drastically by making nomnigraph a separate target